src/BlockTypes/AbstractProductGrid.php

```diff
@@ -172,6 +181,9 @@ protected function parse_attributes( $attributes ) {
 	 * @return array
 	 */
 	protected function parse_query_args() {
+		// Store the original meta query.
+		$this->meta_query = WC()->query->get_meta_query(); // phpcs:ignore WordPress.DB.SlowDBQuery
```

Comment:
Can we remove the phpcs:ignore comment?

Reply:
We sure can, with no complaints from PHPCS.